docs/application/vstools/install.md

```diff
@@ -1,170 +1,104 @@
 
-The Visual Studio Tools for Tizen extension is registered in the Visual Studio Marketplace. You can install extensions from the Visual Studio Marketplace in the Visual Studio IDE.
+1. In the Visual Studio menu, select **Tools > Extensions > Manage Extensions**.
+2. In the **Visual Studio Marketplace**, search for **Tizen**.  
+3. In the **Visual Studio Tools for Tizen** option that appears, click **Download** and close the Visual Studio IDE.
 
-1. In the Visual Studio IDE menu, go to **Tools &gt; Extensions and Updates**.
-2. In the Visual Studio Marketplace, search for **Tizen**.
+![Visual Studio Tools for Tizen](media/v1-install-tools.gif)
 
-   ![Marketplace](media/howtoinstall-extensionwindows.png)
+Alternatively, you can download the Visual Studio Tools for Tizen extension from [Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=tizen.VisualStudioToolsforTizen).
 
-3. Click **Download** and close the Visual Studio IDE.
+## Install Tizen Baseline SDK
 
-   The installation starts.
+Baseline SDK contains Tizen-specific libraries and tools such as profiler, debugger, and so on.
 
-Alternatively, you can download the Visual Studio Tools for Tizen extension from the [Visual Studio Marketplace Web site](https://marketplace.visualstudio.com/items?itemName=tizen.VisualStudioToolsforTizen).
+After installing the Visual Studio Tools for Tizen extension, you must set up the Tizen baseline SDK. You can either install a new Tizen baseline SDK or configure an existing Tizen baseline SDK:
 
-### Installing the Tizen Baseline SDK
+- [Install a new Tizen baseline SDK](#install-new-tizen-baseline-sdk)
+- [Configure an existing Tizen baseline SDK](#configure-existing-tizen-baseline-sdk)
 
-After installing the Visual Studio Tools for Tizen extension, you must set up the Tizen baseline SDK:
+### Install New Tizen Baseline SDK
 
-1. In the Visual Studio IDE menu, go to **Tools &gt; Tizen &gt; Tizen Package Manager**.
+1. In the Visual Studio menu, select **Tools > Tizen > Tizen Package Manager**.
 2. Select **Install new Tizen SDK**.
-
-   ![Select new installation](media/howtoinstall-installwizard1.png)
-
 3. Read the license document and click **I Agree**.
+4. Enter the root directory path where you want to install the Tizen baseline SDK and click **Next**. \
+   The baseline SDK is installed automatically.
+5. In the **Package Manager** window that appears, select **Main SDK**.
+6. Install **Tizen SDK tools**.
 
-   ![Agree to license details](media/howtoinstall-installwizard2.png)
-
-4. Enter the root directory path where you want to install and click **Next**.
-
-   ![Set the installation path](media/howtoinstall-installwizard3.png)
-
-   The Tizen SDK installer is downloaded and the baseline SDK is installed automatically.
-
-   ![Installer download](media/howtoinstall-installwizard4.png)
+![Visual Studio Tools for Tizen](media/v2-install-baseline-sdk.gif)
```

Comment:
The width of the screenshots is too big. Please be shorten the width for Docs site.

Reply:
@journey2w,

Have reduced the width of all the Gifs. If we reduce the width further, the GIF is not clear. Please let us know if the updated GIFs width are fine.